tests/samples/songkick/elysianfields.json

```diff
@@ -261,5 +261,19 @@
                 }
             ]
         }
-    ]
+    ],
+    "dublincore": [
+        {
+            "namespaces": {
+        },
```

Comment:
the indentation looks off: this brace is a closing brace for a brace after "namespaces", not for a brace which is on top of it at the same level

Reply:
done